apps/angular/crud/src/app/state/effects/todo.effects.ts

```diff
@@ -13,13 +19,10 @@ export class TodoEffects {
 
   loadTodos$ = createEffect(() =>
     this.actions$.pipe(
-      ofType(TodoActionName.callTodoList),
+      ofType(TodoActions.callTodoList),
       exhaustMap(() =>
         this.todoService.callTodoList().pipe(
-          map((todoList) => ({
-            type: TodoActionName.loadTodoList,
-            todoList: todoList,
-          })),
+          map((todoList) => TodoActions.loadTodoList({ todoList: todoList })),
           catchError(() => EMPTY),
```

Comment:
never throw EMPTY. You should return an action even if you don't use it. If you have to debug this, you will see the action inside the debugger panel